components/ShowCharacter.js

```diff
@@ -28,7 +28,7 @@ const StyledCard = styled.div`
   max-width: 350px;
   white-space: pre-line;
   word-break: break-word;
-  margin: 20px;
+  margin: 30px 20px 0 20px;
```

Comment:
Just a hint: you can use a shorthand here for /* top | horizontal | bottom */

```suggestion
  margin: 30px 20px 0;
```

Reply:
At this point I still have to screw a bit because of the visibility but I remember the shorthand :)